src/WinRT.Runtime/Projections/IEnumerable.net5.cs

```diff
@@ -998,6 +998,11 @@ public uint GetMany(ref T[] items)
             public bool MoveNext() => _MoveNext();
 
             uint global::Microsoft.UI.Xaml.Interop.IBindableIterator.GetMany(ref object[] items)
+            {
+                // Should not be called.
+                throw new NotImplementedException();
```

Comment:
Nit (a bit of a side note): should these be `NotSupportedException`? IIRC the recommendation was to never throw `NotImplementedException` from production code.

Reply:
I think the `E_NOTIMPL` HResult here actually better matches the expected experience on the WinRT consumption side than throwing a `NotSupportedException` would as this method is only used when a WinRT consumer reinterpet_casts this type to `IIterator<object>`, which is technically invalid, but we need to support.

Reply:
Oh, I see, that makes sense. Thank you for the additional info! 🙂